client/state/billing-transactions/individual-transactions/actions.js

```diff
@@ -0,0 +1,42 @@
+/** @format */
+/**
+ * Internal dependencies
+ */
+import {
+	BILLING_TRANSACTION_REQUEST,
+	BILLING_TRANSACTION_REQUEST_FAILURE,
+	BILLING_TRANSACTION_REQUEST_SUCCESS,
+} from 'state/action-types';
+import wp from 'lib/wp';
+
+export const requestBillingTransaction = transactionId => dispatch => {
+	dispatch( {
+		type: BILLING_TRANSACTION_REQUEST,
+		transactionId,
+	} );
+
+	return wp
+		.undocumented()
+		.me()
+		.getReceipt( transactionId, { format: 'display' } )
+		.then( receipt => {
+			dispatch( {
+				type: BILLING_TRANSACTION_REQUEST_SUCCESS,
+				transactionId,
+				receipt,
+			} );
+		} )
+		.catch( error => {
+			dispatch( {
+				type: BILLING_TRANSACTION_REQUEST_FAILURE,
+				transactionId,
+				error,
+			} );
+		} );
+};
+
+export const clearBillingTransactionError = receiptId => ( {
+	type: BILLING_TRANSACTION_REQUEST_FAILURE,
```

Comment:
It feels a little odd to clear the transaction error with a `BILLING_TRANSACTION_REQUEST_FAILURE` action. I'd expect us to call a different action here, as this doesn't really correspond to a request failure.